src/olympia/hero/tests/test_models.py

```diff
@@ -198,6 +198,22 @@ def test_clean_cta(self):
         hero.cta_url = 'http://goo.gl'
         hero.clean()  # it raises if there's an error
 
+    def test_clean_cta_remove_prefixes(self):
+        hero = SecondaryHero.objects.create()
+
+        with self.activate(locale='en-US', app='firefox'):
+            hero.cta_url = '/en-US/firefox/addon/foo'
+            hero.clean()
+            assert hero.cta_url == '/addon/foo/'
+
+            hero.cta_url = '/en-US/firefox/addon/foo/'
+            hero.clean()
+            assert hero.cta_url == '/addon/foo/'
+
```

Comment:
Do we need tests for a url that's missing the app or locale?

Reply:
Yeah. Added a few more tests in 9bf8b1251a2a22b9bcadeb3bf142c8207ccdbbd3